frontend/src/citizen-frontend/calendar/CalendarPage.tsx

```diff
@@ -98,6 +98,26 @@ const CalendarPage = React.memo(function CalendarPage() {
     [activeFixedPeriodQuestionnaire]
   )
 
+  const calculateFirstReservableDate: () => LocalDate = useCallback(() => {
+    if (data.isSuccess) {
+      const earliesReservableDate =
+        data.value.reservableDays[0].start.isEqualOrAfter(LocalDate.today())
```

Comment:
Dont remember the context but is it possible that `reservableDays` is empty array and `reservableDays[0].start` throws?